### ISSUE TITLE
Add the current tariff indicator

### ISSUE THREAD

**Issue by gmkey** [mschlenstedt/LoxBerry-Plugin-Smartmeter#27]
Hello Michael (et al),

I'm an avid user of the Loxberry SmartMeter plugin. In the dutch smr documentation there is a field which provides valuable information for those using different rates (peak/low). 

The Specification document (see link below) provides the following description. 
 
| Value | OBIS reference | Attribute | Class ID | Value Format | Value Unit |
| --------------- | --------------- | --------------- | --------------- | --------------- | --------------- |
| Tariff indicator electricity. The tariff indicator can also be used to switch tariff dependent loads e.g boilers. This is the responsibility of the P1 user. | 0-0:96.14.0.255  |  2 Value | 1 Data | S4, tag 9 |  |

source: [P1 Companion Standard - Dutch Smart Meter Requirements](https://www.netbeheernederland.nl/_upload/Files/Slimme_meter_15_a727fce1f1.pdf)

This field and values would allow me to trigger certain processes in my loxone based on the current energy tariff. 

I had a look at the code, but I can't figure out the hex-values in the array in the [php_sml_parser.class.php](https://github.com/mschlenstedt/LoxBerry-Plugin-Smartmeter/blob/master/bin/php_sml_parser.class.php).

Would it be possible to add this to the plugin?

**Comment by mschlenstedt** [comment]
One of the next things I have on my todo list is, to switch over zu vzlogger for the plugin: https://wiki.volkszaehler.org/software/controller/vzlogger

vzlogger is much more powerful in reading smartmeters than our own routine. And you can define whatever you want to read from the smartmeter. All what your smartmeter is sending, is also available. So this will definetly solve your issue...

I think a first version will be available in the next few weeks.

**Comment by gmkey** [comment]
Any progress made?

**Comment by mschlenstedt** [comment]
No, never dived deeper into it. I installed a 3-phase meter right behind my official meter with Modbus/TCP (150 EUR) and get rid of these damn incompatibilities every manucacturer includes into their products. Every meter change is like russian roulette - I was suck of all these time I had to invest to get it running back after a meter change.

But there are also good alternative IR reading heads like the Tibber Pulse, which can handle a lot of meters (you do not need a Tibber account to use it with LoxBerry).

So no more enhancements from my side to this plugin - sorry for that.
